src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
+import { routes } from './app.routes';
```

Comment:
suggestion

          import { routes } from './app.routes'; // Importing application routes

          Explantion: Comments on imported routes can provide context on what routes are being used in the application.